scarb/tests/build.rs

```diff
@@ -528,10 +528,16 @@ fn sierra_replace_ids() {
         .assert()
         .success();
 
-    t.child("target/dev/hello.sierra")
-        .assert(predicates::str::contains(
-            "hello::example@0() -> (felt252);",
-        ));
+    t.child("target/dev/hello.sierra").assert(
```

Comment:
We need to discuss the extension with Cairo guys and ensure everybody likes it. Can you please start the conversation on our shared channel @MrDenkoV?